configs/icds-14lakh.yml

```diff
@@ -406,7 +405,7 @@ services:
       ram_static_baseline: 1  # per node
 
   pg_ucr:
-    usage_capacity_per_node: 50000
+    usage_capacity_per_node: 100000
```

Comment:
this is also limited by the disk performance which isn't really captures in this tool at all. If we double the usage we'd need to double to disk performance as well (until we have citus rolled out)